examples/notebook.ipynb

```diff
@@ -128,23 +216,55 @@
+     ]
+    }
+   ],
    "source": [
-    "HOME = [0, -1.571, 1.571, -1.571, -1.571, 0]\n",
+    "async with controller[0] as motion_group:\n",
+    "    home_joints = (await motion_group.joints(\"Flange\")).joints\n",
+    "    current_pose = await motion_group.tcp_pose(\"Flange\")\n",
+    "    target_pose = current_pose @ Pose((1, 0, 0, 0, 0, 0))\n",
```

Comment:
More lib related, but if would prefer an explicit `.transform()` instead of "`@`"

Reply:
Good point, but I would not do it in the MR. I've created a ticket for it https://wandelbots.atlassian.net/browse/RPS-1018. (In the future we need to track issues in Github I guess.)